diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -178,6 +178,24 @@ def custom_error(self, msg):
         dest="plot",
         help="Do not show the plot.",
     )
+    group.add_option(
+        '--usefilenames',
+        action="store_true",
+        dest="usefilenames",
+        help="Use the file names as labels on plot."
```

Comment:
isn't this the default behavior?  So we don't need an option for this, just do this if one of the other options is not selected?

Reply:
default behavior is to use "target" and "morph" as names

Reply:
let's revisit this.  I will merge this and we can have a new quick issue after some discussion what is the best design